src/stage/index.js

```diff
@@ -107,7 +107,10 @@ class Stage extends React.Component {
   componentDidMount() {
     const { onMount, width, height, options, raf } = this.props
 
-    this.app = new PIXI.Application(width, height, {
+    this.app = new PIXI.Application({
+      // TODO: ask @inlet if we should move this to options props as PIXI does?
```

Comment:
@inlet

Reply:
Good point, I think we can get rid of the options prop entirely and pass in any Application option directly as prop. But I'll fix that after merging this PR